docs/reference/search/search-your-data.asciidoc

```diff
@@ -24,55 +24,29 @@ a specific number of results.
 [[run-an-es-search]]
 == Run a search
 
-You can use the <<search-search,search API>> to search data stored in
-{es} data streams or indices.
-
-The API can run two types of searches, depending on how you provide
-queries:
-
-<<run-uri-search,URI searches>>::
-  Queries are provided through a query parameter. URI searches tend to be
-  simpler and best suited for testing.
-
-<<run-request-body-search,Request body searches>>::
-  Queries are provided through the JSON body of the API request. These queries
-  are written in <<query-dsl,Query DSL>>. We recommend using request body
-  searches in most production use cases.
-
-[WARNING]
-====
-If you specify a query in both the URI and request body, the search API request
-runs only the URI query.
-====
-
-[discrete]
-[[run-uri-search]]
-=== Run a URI search
-
-You can use the search API's <<search-api-query-params-q,`q` query string
-parameter>> to run a search in the request's URI. The `q` parameter only accepts
-queries written in Lucene's <<query-string-syntax,query string syntax>>.
+You can use the <<search-search,search API>> to search data stored in {es} data
+streams or indices. The API's `query` request body parameter accepts queries
```

Comment:
Should we link to the aggregation docs here? I think of aggregations as being equally central to a search request as queries.